doc/nrf/ug_thread_certification.rst

```diff
@@ -77,11 +77,37 @@ Complete the following steps to prepare for the certification tests:
 
 #. Prepare Thread Test Harness.
 
-   a. Copy the provided :file:`ncs/nrf/samples/openthread/cli/harness/nRF_Connect_SDK.py` file into :file:`C:\\GRL\\Thread1.1\\Thread_Harness\\THCI\\`.
+   a. Copy the provided :file:`ncs/modules/lib/openthread/tools/harness-thci/OpenThread.py` file into :file:`C:\\GRL\\Thread1.1\\Thread_Harness\\THCI\\nRF_Connect_SDK.py`.
 
    b. Copy the provided :file:`ncs/nrf/samples/openthread/cli/harness/nRF_Connect_SDK.jpg` file into :file:`C:\\GRL\\Thread1.1\\Web\\images\\`.
 
-   c. Edit :file:`C:\\GRL\\Thread1.1\\Web\\data\\deviceInputFields.xml` and prepend the following code:
+   c. Edit :file:`C:\\GRL\\Thread1.1\\Thread_Harness\\THCI\\nRF_Connect_SDK.py` as follows:
+
+      .. code-block::
+         """
+            >> Thread Host Controller Interface
+            >> Device : OpenThread THCI
+            >> Class : OpenThread
+         """
+
+      to
+
+      .. code-block::
+         """
+            >> Thread Host Controller Interface
+            >> Device : nRF_Connect_SDK THCI
+            >> Class : nRF_Connect_SDK
+         """
+
```

Comment:
```suggestion

      and

```
I think that'll make it a little clearer, but will need to see the built documentation to make sure if this needs further clarification.

Reply:
fixed